example/pubspec.lock

```diff
@@ -42,7 +42,7 @@ packages:
       name: collection
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "1.16.0"
+    version: "1.15.0"
```

Comment:
Why it is necessary to decrease package version?
Seems like you committed your local saved versions.